tests/NCronJob.Tests/TestHelper.cs

```diff
@@ -189,3 +193,23 @@ private async IAsyncEnumerable<object> GetCompletionJobsAsync(int expectedJobCou
         }
     }
 }
+
+public sealed class Storage
+{
+#if NET9_0_OR_GREATER
+        private readonly Lock locker = new();
```

Comment:
I am at least 1421 years old and that is why whitespaces peek my interest :D - and of course this is a nitpick. Ignore me :D
```suggestion
    private readonly Lock locker = new();
```

Reply:
That's a good age. It provides with a lot of perspective...

...and fixed